File: noor-shield-pc/service/supabaseConfig.js
'use strict';

/**
 * Shared Supabase project identity for the cloud-backed pieces of Noor
 * Shield (remote control in cloudSync.js, product-key activation in
 * license.js). Safe to ship in the app — see cloud/schema.sql's top comment
 * for why the anon/publishable key grants no more than any anonymous
 * visitor already has; real access control lives in the database itself.
 */

const SUPABASE_URL = 'https://wjqxbjcmcrjgxnxcowxp.supabase.co';
const SUPABASE_ANON_KEY = 'sb_publishable_J-N2ozRX6Rnk1-MI66ORGg_QXG3zJhV';

module.exports = { SUPABASE_URL, SUPABASE_ANON_KEY };

File: noor-shield-parent-app/app/src/main/java/com/barnoota/noorshieldparent/SupabaseConfig.kt
package com.barnoota.noorshieldparent

/**
 * Same Supabase project the PC app and web dashboard use (see
 * noor-shield-pc/service/supabaseConfig.js and cloud/schema.sql) — one
 * shared backend behind the Windows app, the web dashboard, and this phone
 * app. Safe to ship: the anon/publishable key grants no more than any
 * anonymous visitor already has; real access control lives in Supabase's
 * Row Level Security and the device_secret checks inside each
 * SECURITY DEFINER function, not in keeping this key secret.
 */
object SupabaseConfig {
    const val URL = "https://wjqxbjcmcrjgxnxcowxp.supabase.co"
    const val ANON_KEY = "sb_publishable_J-N2ozRX6Rnk1-MI66ORGg_QXG3zJhV"
}
